src/grt/include/grt/GlobalRouter.h

```diff
@@ -361,6 +362,7 @@ class GlobalRouter
   std::vector<RegionAdjustment> region_adjustments_;
 
   int verbose_;
```

Comment:
silent would be a more appropriate
strange to have both verbose and silence.
verbose needs to be renamed and/or coordinated with silent